uk.co.compucorp.civicrm.civihrcore/CRM/CiviHRCore.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class CRM_CiviHRCore {
+  /**
+   * Get version value from info.xml. This method uses caching so it reads
+   * xml file only once during CiviCRM session.
+   *
+   * @return string
+   */
+  public static function getVersion() {
+
```

Comment:
Since noone bothered you about this:
```php
public static function getVersion() {
  $version = CRM_Core_BAO_Cache::getItem('CiviHRCore', 'version');

  if (empty($version)) {
    // ...
  }

  return $version;
}
```
let the code breathe a little